crates/exex/exex/src/lib.rs

```diff
@@ -49,3 +49,18 @@ pub use manager::*;
 // Re-export exex types
 #[doc(inline)]
 pub use reth_exex_types::*;
+
+use reth_chainspec::Head;
+use reth_primitives::{BlockHash, BlockNumber};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct ExExHead {
+    pub number: BlockNumber,
+    pub hash: BlockHash,
+}
```

Comment:
this needs proper docs,

do we think we will eventually require additional info from the exex?
if so we could perhaps change this to

```rust
struct ExExInfo {
   head: Option<ExExHead>
}
```